rule/constant-logical-expr.go

```diff
@@ -72,7 +72,7 @@ func (*lintConstantLogicalExpr) isOperatorWithLogicalResult(t token.Token) bool
 	return false
 }
 
-func (*lintConstantLogicalExpr) isEqualityOperator(t token.Token) bool {
+func (l *lintConstantLogicalExpr) isEqualityOperator(t token.Token) bool {
```

Comment:
`l` is unused here. Please check other places where it is unused. This will make the PR smaller.